lib/modules/datasource/conda/prefix-dev.ts

```diff
@@ -0,0 +1,159 @@
+import is from '@sindresorhus/is';
+import { z } from 'zod';
+import { logger } from '../../../logger';
+import { isNotNullOrUndefined } from '../../../util/array';
+import type { Http } from '../../../util/http';
+import type { Timestamp } from '../../../util/timestamp';
+import { MaybeTimestamp } from '../../../util/timestamp';
+import type { ReleaseResult } from '../types';
+
+const MAX_PREFIX_DEV_GRAPHQL_PAGE = 100;
+
+const File = z.object({
```

Comment:
schemas should be in a `schema.ts` file

Reply:
I moved them here beacuse they are only used in this file , I'll move them back